src/std/fs.ab

```diff
@@ -76,3 +78,45 @@ pub fun change_owner(user: Text, path: Text): Bool {
 
     return false
 }
+
+/// Finds all files or directories matching a file glob.  Escapes all
+/// characters in the passed-in glob apart from "*", "?" and "/", to
+/// prevent injection attacks.
+pub fun glob(path: Text): [Text]? {
+    path = replace_regex(path, "\([^*?/]\)", "\\\\\1")
+    let files = $eval "for file in {path}; do [ -e \\\"\\\$file\\\" ] && echo \\\"\\\$file\\\"; done"$?
+    return split(files, "\n")
+}
+
+/// Finds all files or directories matching multiple file globs.  When
+/// we have union types, these two functions can be merged as follows:
+///
+/// ```
+/// pub fun glob(path: Text | [Text]): [Text]? {
+///     if {
+///         path is Text: {
+///             path = replace_regex(path, "\([^*?/]\)", "\\\\\1")
+///             let files = $eval "for file in {path}; do [ -e \\\"\\\$file\\\" ] && echo \\\"\\\$file\\\"; done"$?
+///             return split(files, "\n")
+///         }
+///         path is [Text]: {
+///             let items = [Text]
+///             loop item in path {
+///                 item = replace_regex(item, "\([^*?/]\)", "\\\\\1")
+///                 items += [item]
+///             }
+///             let files = $eval "for file in {items}; do [ -e \\\"\\\$file\\\" ] && echo \\\"\\\$file\\\"; done"$?
+///             return split(files, "\n")
+///         }
+///     }
+/// }
+/// ```
+pub fun glob_multiple(path: [Text]): [Text]? {
```

Comment:
Seems like a good idea to eventually support either `glob("*.txt")` or `glob(["*.txt", "*.xyz"])`.  This function is here as a placeholder for that functionality, so we can write unit tests.  To be merged into the main function when the union types PR is merged.